scripts/scil_surface_apply_transform.py

```diff
@@ -68,26 +71,25 @@ def main():
     args = parser.parse_args()
     logging.getLogger().setLevel(logging.getLevelName(args.verbose))
 
-    assert_inputs_exist(parser, [args.in_surface, args.ants_affine],
-                        args.ants_warp)
+    assert_inputs_exist(parser, [args.in_moving_surface, args.in_transfo],
+                        args.in_deformation)
     assert_outputs_exist(parser, args, args.out_surface)
 
     # Load mesh
-    mesh = load_mesh_from_file(args.in_surface)
+    mesh = load_mesh_from_file(args.in_moving_surface)
 
-    warp_img = None
-    # Load affine
-    affine = np.loadtxt(args.ants_affine)
+    # Load transformation
+    transfo = load_matrix_in_any_format(args.in_transfo)
 
-    # Non linear transformation
-    if args.ants_warp:
-        # Load warp
-        warp_img = nib.load(args.ants_warp)
+    deformation_data = None
+    if args.in_deformation is not None:
+        deformation_data = nib.load(args.in_deformation)
 
-    mesh = apply_transform(mesh, affine, warp_img)
+    mesh = apply_transform(mesh, transfo, deformation_data,
+                           inverse=args.inverse)
 
     # Save mesh
-    mesh.save(args.out_surface)
+    save_polydata(mesh.__polydata__, args.out_surface, legacy_vtk_format=True)
```

Comment:
I think we decided that the only way to save in legacy was to use the convert. The default for all script was to save the new format (even if it does not work in MI-Brain).

I personally recommend using PLY instead (as long as MI-Brain does not update)

Reply:
I don't recall we did say that. Look at scil_surface_convert.py we save it so MI-Brain can read them.